common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -125,6 +125,12 @@
     ],
     "sqlState" : "22003"
   },
+  "BOOLEAN_STATEMENT_EMPTY_ROWS" : {
+    "message" : [
+      "Boolean statement <invalidStatement> is invalid. It returns an empty row."
```

Comment:
Let's make this error text more precise like:
`Boolean statement <invalidStatement> is invalid. Expected single row with a value of the BOOLEAN type, but got an empty row.`

Reply:
Updated.